### ISSUE TITLE
Cannot pull image on AKS

### ISSUE THREAD

**Issue by bukton123** [acorn-io/acorn#721]
I can run acorn, but I see the problem pull image on the pod

**Using AKS**
```
STATUS: ENDPOINTS[] HEALTHY[0/2] UPTODATE[1/2] [containers: fragrant-fire is not ready; survey ErrImagePull: rpc error: code = Unknown desc = failed to pull and unpack image 

"127.0.0.1:32257/acorn/acorn@sha256:4732fbf30a7a20805be784b71b36ed84a49d4c4fc8c9a2e0d237401d1fa51042": failed to resolve reference 

"127.0.0.1:32257/acorn/acorn@sha256:4732fbf30a7a20805be784b71b36ed84a49d4c4fc8c9a2e0d237401d1fa51042": 
failed to do request: Head 

"https://127.0.0.1:32257/v2/acorn/acorn/manifests/sha256:4732fbf30a7a20805be784b71b36ed84a49d4c4fc8c9a2e0d237401d1fa51042": http: server gave HTTP response to HTTPS client
```

**Comment by cjellick** [comment]
@meldafrawi is this the AKS issue you've been working on?

**Comment by meldafrawi** [comment]
Yes, it is the same one.
Ref: #689

**Comment by meldafrawi** [comment]
@bukton123 issue is fixed in v0.3.0 release.

**Comment by bukton123** [comment]
@meldafrawi Thank you
